src/uikit.cs

```diff
@@ -14159,12 +14159,12 @@ interface UIViewController : NSCoding, UIAppearanceContainer, UIContentContainer
 
 		[iOS (11,0), TV (11,0)]
 		[Export ("viewLayoutMarginsDidChange")]
-		[Advice ("You must call the base method when overriding.")] // [RequiresSuper]
+		[RequiresSuper]
 		void ViewLayoutMarginsDidChange ();
 
 		[iOS (11,0), TV (11,0)]
 		[Export ("viewSafeAreaInsetsDidChange")]
-		[Advice ("You must call the base method when overriding.")] // [RequiresSuper]
+		[RequiresSuper]
```

Comment:
ran `xtro` ?
because if those were not reported before it's either a bug (in xtro) or those are not decorated in the headers (and the data file will need an update)

Reply:
Ah no I did not, I assumed the comment was right (👍 
Will do.